packages/OpenSqueakMap.package/OSMRenderConfigurationBrowser.class/instance/setCurrentValueFor.To..st

```diff
@@ -0,0 +1,14 @@
+update
+setCurrentValueFor: anAttribute To: aValue
+
+	| renderConfiguration rule |
+	renderConfiguration := self renderConfiguration.
+	rule := renderConfiguration rules at: self selectedType.
+	rule basicAttributes at: anAttribute put: aValue.
+	renderConfiguration at: self selectedType put: rule.
+	
+	self viewport chunkManager 
```

Comment:
Maybe the `RenderConfigurationBrowser` should only know about the `Viewport`, not the `ChunkManager`?

We could add something like `updateRenderConfig` to `Viewport` where the whole updating, applying and invalidating happens